src/routes/docs/tutorials/sveltekit/step-6/+page.markdoc

```diff
@@ -21,6 +21,8 @@ Create a new collection with the following attributes:
 | title       | String | Yes      |
 | description | String | No       |
 
+Setup permissions on your collection to allow all users for now. In case you get 401 error, check this step.
```

Comment:
I would pick a more specific permission and add a table to show the role + permissions.

I would also mention where in the navigation this can be updated :)